frontend/packages/ux-editor/src/data/formItemConfig.ts

```diff
@@ -152,12 +156,39 @@ export const formItemConfigs: FormItemConfigs = {
   [ComponentType.CustomButton]: {
     name: ComponentType.CustomButton,
     itemType: LayoutItemType.Component,
+    getDisplayName: (
+      formItem: ComponentSpecificConfig<ComponentType.CustomButton>,
+    ): ComponentType | CustomComponentType => {
+      const action = formItem?.actions?.length === 1 && formItem?.actions?.[0];
+      return action?.id === 'closeSubform' && action?.type === 'ClientAction'
+        ? CustomComponentType.CloseSubformButton
+        : ComponentType.CustomButton;
+    },
     defaultProperties: {
       actions: [],
       buttonStyle: 'primary',
     },
     icon: FingerButtonIcon,
   },
+  [CustomComponentType.CloseSubformButton]: {
+    name: CustomComponentType.CloseSubformButton,
+    componentRef: ComponentType.CustomButton,
+    itemType: LayoutItemType.Component,
+    defaultProperties: {
+      actions: [
+        {
+          type: 'ClientAction',
+          id: 'closeSubform',
+          validation: {
+            page: 'all',
+            show: ['All'],
```

Comment:
I am not sure 🤔, just wonder de we need this as array here?

Reply:
You mean this: `show: ['All'],`? I think so, it is how team apps has set up in the config for the app 🙂